src/azure-cli/azure/cli/command_modules/acr/commands.py

```diff
@@ -197,12 +205,24 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
     with self.command_group('acr repository', acr_repo_util) as g:
         g.command('list', 'acr_repository_list')
         g.command('show-tags', 'acr_repository_show_tags')
-        g.command('show-manifests', 'acr_repository_show_manifests')
+        g.command('show-manifests', 'acr_repository_show_manifests',
+                  deprecate_info=self.deprecate(redirect='acr manifest metadata list', hide=True))
```

Comment:
create a work item to delete it in 3 months (people tend to forget and leave them there).

Reply:
good idea